### PR TITLE
feat: Enhance observability

### PR DESCRIPTION
**Pull request by jiayouxujin**
Hi,
this pr is for #20

### CODE REVIEW

File: pkg/resources/container.go [hunk 1]
```diff
@@ -27,6 +30,23 @@ func NewInstanceContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
 	return container
 }
 
+func NewExporterContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
+	return &corev1.Container{
+		Name:  "kvrocks-exporter",
+		Image: "jinxu95/kvrocks_exporter:latest",
```
Comment (jiayouxujin) [inline pkg/resources/container.go]: Hi @ColinChamber .
This image needs to be replaced with the official.

Comment (ColinChamber) [reply pkg/resources/container.go]: Please review https://github.com/RocksLabs/kvrocks_exporter/pull/23

File: pkg/resources/container.go [hunk 2]
```diff
@@ -27,6 +30,23 @@ func NewInstanceContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
 	return container
 }
 
+func NewExporterContainer(instance *kvrocksv1alpha1.KVRocks) *corev1.Container {
+	return &corev1.Container{
+		Name:  "kvrocks-exporter",
+		Image: "hulkdev/kvrocks-exporter:latest",
+		Args: []string{
+			fmt.Sprintf("--kvrocks.addr=%s://localhost:%s", instance.Namespace, strconv.Itoa(kvrocks.KVRocksPort)),
```
Comment (ColinChamber) [inline pkg/resources/container.go]: Is the first placeholder replaced with the namespace?

Comment (jiayouxujin) [reply pkg/resources/container.go]: Thanks, I have solved it.

**Review by ColinChamber — APPROVED** [review]

**Pull request merged**